src/util/dates.js

```diff
@@ -0,0 +1,26 @@
+import DATE_RANGE_OPTIONS from "../constants/DateRangeOptions";
+
+export function convertDateRange(selectedDateRange) {
+  switch (true) {
```

Comment:
This is fine as is, since there aren't too many values. If there were more, I'd suggest iterating over the `DATE_RANGE_OPTIONS` and building a map of display text to query text.

Reply:
Good suggestion, for sure

Reply:
Done